cmd/snap/cmd_snap_op_test.go

```diff
@@ -1302,6 +1292,22 @@ func (s *SnapOpSuite) TestRefreshOneRebooting(c *check.C) {
 
 }
 
+func (s *SnapOpSuite) TestRefreshOneChanDeprecated(c *check.C) {
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, `{"type": "error", "result": {"message": "snap not found", "value": "foo", "kind": "snap-not-found"}, "status-code": 404}`)
+	})
+
+	for in, out := range map[string]string{
+		"/foo":    "foo/stable",
+		"/stable": "latest/stable",
```

Comment:
Should we add something like `		"///foo/stable//": "foo/stable",` here too ? I mean, something that also has extra slashes at the end or something like this

Reply:
done